<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>587</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_op1_text">
      <property name="text">
       <string>Операнд 1</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1" colspan="2">
     <widget class="QPushButton" name="button_answer">
      <property name="text">
       <string>Выполнить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_op2_text">
      <property name="text">
       <string>Операнд 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="1">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QRadioButton" name="button_mins">
        <property name="text">
         <string>Вычитание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="button_multi">
        <property name="text">
         <string>Умножение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="button_divide">
        <property name="text">
         <string>Деление</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="button_upper">
        <property name="text">
         <string>Возведение в степень</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="button_dupper">
        <property name="text">
         <string>Корень</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="button_sin">
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="button_arcsin">
        <property name="text">
         <string>arcsin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="button_cos">
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="button_arccos">
        <property name="text">
         <string>arccos</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="button_tg">
        <property name="text">
         <string>tg</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="button_ctg">
        <property name="text">
         <string>ctg</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="button_plus">
        <property name="text">
         <string>Сложение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="1">
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="label_percent">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_result">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	backgrouncolor = white;
}</string>
        </property>
        <property name="text">
         <string>Результат:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_resultAnswer">
        <property name="text">
         <string>Значение</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_result_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	backgrouncolor = white;
}</string>
        </property>
        <property name="text">
         <string>Остаток от деления:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
